assessments/guide/security-reviewer.md

```diff
@@ -43,4 +43,31 @@ expected to conduct an in-depth review described in the [Security Assessment
 Guide](./). Security Reviewers should seek the approval of the other
 participating Security Reviewers and at least 1 co-chair before merging.
 
+## Conflict of interest
 
+There is a possibility of conflict of interest that can arise between a security reviewer and
+the project being reviewed due to the closely-knit nature of the community. Having clear
+guidelines for conflict of interest situations are important to prevent:
+
+- Individuals from intentionally or unintentionally promote their own company's project
+- Sig-security chairs and assessment leads could intentionally or unintentionally limit the participation of an individual unfairly by asserting conflict of interest
+- Security reviews being stalled while groups discuss whether this person or that person should be allowed to participate
+
+The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts.
+Hard conflicts makes a reviewer ineligible to review a project.
+A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
+A reviewer with a soft conflict is required to document the nature of the conflict in the
+security assessment documentation. It is the responsibility of the security reviewer
+to make known his/her own conflict of interests.
+
+Below are classifications of hard/soft conflicts.
+
+Hard conflicts:
+- Reviewer is a maintainer of the project
+- Reviewer is a direct report of/to a maintainer of the project
+- Reviewer is paid to work on the project
+
+Soft conflicts:
+- Reviewer belongs to the same company/organization of the project, but does not work on the project
+- Reviewer uses the project in his/her work
```

Comment:
- Reviewer has contributed to the project.

both this and using the project could be a strong benefit.